src/views/Layout.jsx

```diff
@@ -1,31 +1,54 @@
 import { Outlet, NavLink } from 'react-router-dom';
 
 import './Layout.css';
+import logo from '../../public/img/logo-bluebg.png';
```

Comment:
usually it's best practice if an image is imported into code to have it somewhere in the src dir instead of the public dir. The main advantage, I think, is that if the image is accidentally removed if it's in src it'll create a build-time error, while if it were in public, it wouldn't and just be a broken image

Reply:
Good to know! I've moved the logo image to `src/assets` folder